stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messages/QuotedMessageContent.kt

```diff
@@ -20,32 +20,50 @@ import androidx.compose.foundation.layout.Row
 import androidx.compose.runtime.Composable
 import androidx.compose.ui.Modifier
 import com.getstream.sdk.chat.utils.extensions.isMine
+import io.getstream.chat.android.client.ChatClient
 import io.getstream.chat.android.client.models.Message
 import io.getstream.chat.android.compose.ui.attachments.content.QuotedMessageAttachmentContent
 import io.getstream.chat.android.compose.ui.theme.ChatTheme
 
 /**
  * Represents the default quoted message content that shows an attachment preview, if available, and the message text.
  *
- * @param message The message to show.
+ * @param message The quoted message to show.
  * @param modifier Modifier for styling.
+ * @param replyMessage The message that contains the reply.
  * @param attachmentContent The content for the attachment preview, if available.
  * @param textContent The content for the text preview, or the attachment name or type.
  */
 @Composable
 public fun QuotedMessageContent(
     message: Message,
     modifier: Modifier = Modifier,
+    replyMessage: Message? = null,
     attachmentContent: @Composable (Message) -> Unit = { DefaultQuotedMessageAttachmentContent(it) },
-    textContent: @Composable (Message) -> Unit = { DefaultQuotedMessageTextContent(it) },
+    textContent: @Composable (Message) -> Unit = {
+        DefaultQuotedMessageTextContent(
+            message = it,
+            replyMessage = replyMessage,
+        )
+    },
 ) {
-    val isMyMessage = message.isMine()
+    val messageBubbleShape = if (message.isMine(ChatClient.instance())) {
+        ChatTheme.shapes.myMessageBubble
+    } else {
+        ChatTheme.shapes.otherMessageBubble
+    }
 
-    val messageBubbleShape = if (isMyMessage) ChatTheme.shapes.myMessageBubble else ChatTheme.shapes.otherMessageBubble
+    // The quoted section color depends on the author of the reply.
+    val messageBubbleColor = if (replyMessage?.isMine(ChatClient.instance()) != false) {
```

Comment:
Could we safely flip this to:

```kotlin
    val messageBubbleColor = if (replyMessage?.isMine(ChatClient.instance()) == true) {
```

Reply:
Want to treat "null" replyMessage as "true".